thumbor/handlers/__init__.py

```diff
@@ -281,7 +281,7 @@ def finish_request(self, context, result_from_storage=None):
             context.config.RESULT_STORAGE_STORES_UNSAFE or not context.request.unsafe)
 
         def inner(future):
-            results, content_type = future.result()
+            results, content_type = (future.result()[0], BaseEngine.get_mimetype(future.result()[0]))
```

Comment:
It would be awesome to allow plugins to register new mime types. This was something we wanted to accomplish in thumbor-community/core

If we had a main static class for the mimetype library plugins could do something like:
```
from thumbor import MimeTypes

MimeTypes.register('image/gif', '.gif')
```

Instead of relying on a static list.